### DISCUSSION TITLE
RUB Login via SSO

### DISCUSSION THREAD

**Discussion by BizarrePenguin** [astarub/campus_app#54] [General]
I noticed that there are preparations in the code for a future login with loginID and password for some services. I just wondered if there are plans to implement it using the SSO login that the RUB provides. I think that it would create a unified experience for the users.

**Comment by henry-herrmann** [comment]
We haven't agreed on any details regarding that and it's currently not on our agenda as specific requirements have to be sorted out first.

**Comment by domai-tb** [comment]
Yes, we already prepared a SSO login based on the RUB ForeRock instance, but like Henry said, it is not on our coding agenda rith now. The code is from a very alpha stage of this app and based on reverse engineering. 

On the other hand we are ready to start discussions with the university to get access to the official ForeRock / eCampus API. Last week we wrote an E-Mail to the rectoral office were we asking for a meeting about support by the university authorities. We hope to get a date in january next year.  So we will announce updates about that as soon as we can. :)

Cheers! :Domai

**Reply by Nicole4Fox** [reply]
Anything new on this?

**Reply by henry-herrmann** [reply]
Negative, will probably be implemented in version 2.3.
